shared/controlflow/change-notes/2025-01-16-basic-block.md

```diff
@@ -0,0 +1,7 @@
+---
+category: breaking
+---
+* Added a basic block construction as part of the library. This is currently
+  considered an internal unstable API. The input signature to the control flow
+  graph now requires two additional predicates: `idOfAstNode` and
+  `idOfCfgScope`.
```

Comment:
The implementation still has `default` empty implementations, which should be removed.